charts/incubator/plausible/values.yaml

```diff
@@ -1,28 +1,26 @@
 image:
-  repository: plausible/analytics
+  repository: tccr.io/truecharts/plausible
   pullPolicy: IfNotPresent
-  tag: v1.5.1@sha256:3fbdcfac56ffa79e778e9e4bc1ca8c4dca0d7deb330ed5694f59eaa1125cc0a8
+  tag: v2.0.0@sha256:5bf2057948e1d5d12ef61f446c4945a336d3bcdf9b96293a1d6fc275f3c56eb5
 
 # Plausible self-hosting documentation
 # https://plausible.io/docs/self-hosting
 
 plausible:
   # Server
   BASE_URL: http://localhost
-  DISABLE_AUTH: false
-  DISABLE_REGISTRATION: false
-
-  # Default User
-  ADMIN_USER_NAME: admin
-  ADMIN_USER_EMAIL: admin@example.com
-  ADMIN_USER_PWD: P@ssw0rd
+  DISABLE_REGISTRATION: "true"
+  LOG_FAILED_LOGIN_ATTEMPTS: false
+  # todo(this is fine for SCALE, but not for other multi-node platforms)
+  # RELEASE_DISTRIBUTION: none
```

Comment:
You can leave this commented here, but not expose in the UI.
If someone is "advanced" enough, can add it on scale with "Additional envs"

> I have no idea what this does :D

Reply:
From the Plausible docs:

> Alternatively, if you are running on a single machine and don't need any of Erlang's multi-node distribution features, you turn them off entirely by setting environment variable `RELEASE_DISTRIBUTION` to `none`

See also: https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-environment-variables

This would be a optimization to set this to `none` for undistributed deployments. But it's probably best to keep it simple and just leave these advanced configs for advanced users.